sample/sample.c

```diff
@@ -14,20 +16,30 @@ double delay() {
     return j;
 }
 
+int sig = 0;
+
+/// Allow to close cleanly with ctrl + c
+void sigintHandler(int sig_num) {
+    sig = sig_num;
+    printf("Interrupted\n");
+}
+
+int main( ) {
+    signal(SIGINT, sigintHandler);
```

Comment:
This is building on Windows in Visual Studio - I confess I have not tried it on other platforms yet.